docs/deployment/forked_threaded_servers.rst

```diff
@@ -0,0 +1,87 @@
+Forked and Threaded Servers
++++++++++++++++++++++++++++
+
+Forked and Threaded Servers share common "gotchas" and solutions when
+using pyramid and some popular packages.
+
+Forked and Threaded servers tend to use a "copy on write" implementation detail
+to optimize how they work and share memory. This can create problems when
+certain actions happen before the fork or thread dispatch, such as when files or
+file-descriptors are opened or random number generators are initialized.
+
+Many servers have
```

Comment:
Many servers have what?

Reply:
grrr. replaced from memory.  i tripped on a dog and spilled coffee on my keyboard this morning; switched to the backup keyboard and i've managed to accidentally cut/paste/delete/etc nonstop all day.